lib/init/passport.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * @fileoverview Passport authentication init
+ */
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const database = require('../database');
+const bcrypt = require('bcrypt-nodejs');
+
+// Passport Authentication setup
+passport.use(new LocalStrategy(
+  {
+    usernameField: 'email',
+  },
+  (username, password, cb) => {
+    database.select('*').from('users').where({
+      email: username,
+    })
+    .then(user => {
+      if (user.length < 1) {
+        return cb(null, false, { message: 'Email not in system' });
+      }
+
+      if (!bcrypt.compareSync(password, user[0].password)) {
+        return cb(null, false, { message: 'Incorrect password' });
+      }
```

Comment:
From a [security perspective](http://ux.stackexchange.com/questions/13516/how-to-tell-the-user-his-login-credentials-are-incorrect) incorrect username and password should have the same messaging. While telling a user that either the email address or password is incorrect, if someone wanted to be malicious, they could keep guessing combos until they got an "incorrect password" error, then they would know the email exists in the system.